src/main/resources/locales/en/sjp-press-list-new-cases.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I wonder the name of this language file should be 'single-justic-procedure-press-new-cases.json' to keep it consistent with the other SJP files?

Reply:
Language file name has been changed